GameDataParser/Parsers/ScriptParser.cs

```diff
@@ -249,14 +249,14 @@ public static List<ScriptMetadata> ParseQuest(MemoryMappedFile m2dFile, IEnumera
 
         public static void Write(List<ScriptMetadata> entities)
         {
-            using (FileStream writeStream = File.Create(VariableDefines.OUTPUT + "ms2-script-metadata"))
+            using (FileStream writeStream = File.Create($"{Paths.OUTPUT}/ms2-script-metadata"))
             {
                 Serializer.Serialize(writeStream, entities);
             }
-            using (FileStream readStream = File.OpenRead(VariableDefines.OUTPUT + "ms2-script-metadata"))
+            using (FileStream readStream = File.OpenRead($"{Paths.OUTPUT}/ms2-script-metadata"))
             {
             }
-            Console.WriteLine("Successfully parsed script metadata!");
+            Console.WriteLine("\rSuccessfully parsed script metadata!");
```

Comment:
`Environment.NewLine` is preferable here

https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-5.0